src/plugins/data/common/field_formats/field_format.ts

```diff
@@ -68,7 +73,16 @@ export abstract class FieldFormat {
    */
   public type: any = this.constructor;
 
-  constructor(public _params: any = {}) {}
+  public _params: any;
```

Comment:
nit: Should this be `private`?  The underscore usually implies that something is private, so I found this a bit confusing

Reply:
Nice catch. Changed